homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -96,3 +112,47 @@ def _async_process_data(self):
             self._icon = FLAME_ICON if self._is_on else IDLE_ICON
 
         self.async_write_ha_state()
+
+
+class PwNotifySensor(PwBinarySensor, BinarySensorEntity):
+    """Representation of a Plugwise Notification binary_sensor."""
+
+    def __init__(self, hass, api, coordinator, name, dev_id, binary_sensor, model):
```

Comment:
Don't pass in `hass`. It will be set on the entity when the entity has been added to home assistant. This means you can access `hass` with `self.hass`, after the entity has been added to home assistant.

Reply:
Check, will do